.github/workflows/ci.yml

```
@@ -30,10 +30,10 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v1
-    - name: Setup JDK 1.8
+    - name: Setup JDK 11
```

ℹ️ for easiest maintenance
```suggestion
    - name: Setup JDK
```